flybywiresim/a32nx
# feat: enable efb

Fixes #1251

## Summary of Changes

* Enabled React EFB app to display in production builds

### Fixes

* [x] Status bar collapsing on big pages
* [x] Map shows wrong location (API issue - needs changes there)

## Testing instructions


* Make sure the screen is properly usable **(Functionality is almost non-existent at the moment)**
* Make sure there are no visual glitches
* Make sure the tablet turns on with aircraft AC power


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
